src/unix/bsd/freebsdlike/mod.rs

```diff
@@ -13,6 +13,7 @@ pub type speed_t = ::c_uint;
 pub type nl_item = ::c_int;
 pub type id_t = i64;
 pub type vm_size_t = ::uintptr_t;
+pub type timer_t = *mut ::c_void;
```

Comment:
Is `timer_t` `c_void` on DragonFlyBSD as well?
I only found this: https://github.com/DragonFlyBSD/DragonFlyBSD/blob/85f00d18bcc1ab1d11df7193b597e70a43e0896d/sys/cpu/x86_64/include/stdint.h#L125